src/popup/router/pages/Retip.vue

```diff
@@ -83,14 +85,12 @@ export default {
   async created() {
     this.loading = true;
     this.verifiedUrls = (await axios.get(`${BACKEND_URL}/verified`)).data;
-
     await this.$watchUntilTruly(() => this.sdk);
     this.minCallFee = calculateFee(TX_TYPES.contractCall, {
       ...this.sdk.Ae.defaults,
       contractId: this.network[this.current.network].tipContract,
       callerId: this.account.publicKey,
     }).min;
-
```

Comment:
`created` contains unrelated parts, I'm not sure that it is needed to remove these empty lines